std/bigint.d

```diff
@@ -1761,3 +1761,111 @@ unittest
     auto b = immutable BigInt("123");
     assert(b == 123);
 }
+
+/// Extracts the quotient and remainder for the given dividend and divisor.
+void divMod(const BigInt dividend, const BigInt divisor, out BigInt quotient, out BigInt remainder) pure nothrow
+{
+    BigUint q, r;
+    BigUint.divMod(dividend.data, divisor.data, q, r);
+    quotient.sign = dividend.sign != divisor.sign;
+    quotient.data = q;
+    remainder.sign = dividend.sign;
+    remainder.data = r;
+}
+
+/// ditto
+void divMod(T)(const BigInt dividend, T divisor, out BigInt quotient, out BigInt remainder) pure nothrow
```

Comment:
Remove the attributes here, they are automatically inferred for templated functions and by manually specifying them, future improvements which e.g. would make this `@nogc` are overwritten.